### ISSUE TITLE
FormLabel - CSS class "col-form-label" is missing when column attribute is "sm" or "lg"

### ISSUE THREAD

**Issue by sylmarch** [react-bootstrap/react-bootstrap#5381]
## Describe the bug

According to [bootstrap v4.3 documentation](https://getbootstrap.com/docs/4.3/components/forms/#horizontal-form-label-sizing), following classes should be set on `<label>` tag:
- for small size: `col-form-label col-form-label-sm`
- for normal size: `col-form-label`
- for large size: `col-form-label col-form-label-lg`

`<FormLabel>` component does not correctly handle small size and large size when `column` attribute is set to `sm` or `lg`

### Normal size (= OK)
`<FormLabel>My label</FormLabel>`

generates

`<label class="form-label" for="myID">My label</label>`

### Small size (= NOK)

`<FormLabel column="sm" >My label</FormLabel>`

generates:
`<label for="myID" class="form-label col-form-label-sm col">My label</label>`

instead of
`<label for="myID" class="form-label col-form-label col-form-label-sm col">My label</label>`

### Large size (= NOK)

`<FormLabel column="lg" >My label</FormLabel>`

generates:
`<label for="myID" class="form-label col-form-label-lg col">My label</label>`

instead of
`<label for="myID" class="form-label col-form-label col-form-label-lg col">My label</label>`

## Environment

- React-Bootstrap Version 1.3.0

**Comment by taion** [comment]
Use the `column` prop to control this; this isn't relevant except in the row/col context.

**Comment by sylmarch** [comment]
I'm already using `column` prop of `FormLabel` so I don't understand your answer... Could you explain a little bit more?


Besides, I'm in a row/col context. My complete code looks like this:
```html
<Form.Group controlId="firstname" as={Form.Row}>
	<Form.Label column="sm" sm={3}>
		Firstname:
	</Form.Label>
	<Col sm={9}>
		<Form.Control
			type="text"
			size="sm"
			value={firstname}
			onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
				setFirstname(e.currentTarget.value);
			}}
		/>
	</Col>
</Form.Group>
```

It generates the HTML:
```html
<div class="form-group form-row">
    <label for="firstname" class="form-label col-form-label-sm col-sm-3">Firstname:</label>
    <div class="col-sm-9">
        <input type="text" id="firstname" class="form-control form-control-sm" value="">
    </div>
</div>
```

instead of:
```html
<div class="form-group form-row">
    <label for="firstname" class="form-label col-form-label col-form-label-sm col-sm-3">Firstname:</label>
    <div class="col-sm-9">
        <input type="text" id="firstname" class="form-control form-control-sm" value="">
    </div>
</div>
```

From a visual point of view, if you don't set `col-form-label`, the CSS property `margin-bottom` is not set to 0 (Bootstrap CSS class `.col-form-label-sm` overrides all property from `.col-form-label` **except**  `margin-bottom` ). So you are leaving a huge space below the label and you are not "compacting" the form...

Hereby Bootstrap CSS classes:
```css
.col-form-label-sm {
    padding-top: calc(0.25rem + 1px);
    padding-bottom: calc(0.25rem + 1px);
    font-size: 0.875rem;
    line-height: 1.5;
}
```

```css
.col-form-label {
    padding-top: calc(0.375rem + 1px);
    padding-bottom: calc(0.375rem + 1px);
    margin-bottom: 0;
    font-size: inherit;
    line-height: 1.5;
}
```

**Comment by taion** [comment]
ok, i see; i think we just need to add that class, then. feel free to send in a PR to do so
